secure-app-model/keyvault/partnerconsent/src/main/java/com/microsoft/store/samples/partnerconsent/security/KeyVaultProvider.java

```diff
@@ -52,7 +42,7 @@ public KeyVaultProvider(String vaultBaseUrl, String clientId, String clientSecre
      */
     public String getSecret(String secretName)
```

Comment:
Javadoc for this method should say:
Gets the value of the specified secret from the Azure Key Vault.